gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -330,10 +382,36 @@ private MapGeoJson toMapGeoJson(GeoJsonFeature feature) {
   }
 
   public LiveData<CameraPosition> getCameraPosition() {
+    checkPointNearVertex(cameraPosition.getValue());
```

Comment:
This shouldn't belong here. `getCameraPosition` should only be doing what it is defined to do. 
This check should be specifically called from within the polygon drawing flow.

Reply:
ok

Reply:
removed.